code/index.html

```diff
@@ -3,28 +3,347 @@
+          <option 
+          value="fr"
+          class="form-select-box-option">
+          France</option>
+          <option 
+          value="it"
+          class="form-select-box-option">
+          Italy</option>
+          <option 
+          value="au"
+          class="form-select-box-option">
+          Austria</option>
+          <option 
+          value="sw"
+          class="form-select-box-option">
+          Switzerland</option>
+        </select>
       </div>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <div class="form-select-group">
+        <label for="resortType" 
+        class="form-label-heading">
+        Resort type</label>
+
+        <select 
+        name="resortTypeValue" 
+        id="resortType"
+        required
+        class="form-select-box">
+        <option 
+          value=""
+          disabled selected hidden
+          class="form-select-box-option">
+          Choose resort type</option>
+          <option 
+          value="ch"
+          class="form-select-box-option">
+          Chalets</option>
+          <option 
+          value="pa"
+          class="form-select-box-option">
+          Private Apartments</option>
+          <option 
+          value="hr"
+          class="form-select-box-option">
+          Hotel & Resorts</option>
+        </select>
+      </div>
+
+      <!-- group size: radio type (I did not like how it came out visually, so I am hiding this part)   
+      <h3 class="form-label-heading">Group size</h3>
+  
+      <div class="form-size-group">
+        <input 
+        id="oneTwo" 
+        type="radio"
+        required
+        value="one_two"
+        name="groupSizeValue"
+        class="form-input-radio">
+        
+        <span class="form-custom-radio"></span>
+
+        <label 
+        for="oneTwo" 
+        class="form-label-input">
+        1 - 2</label>
+        </div>
+            
+      <div class="form-size-group">
+        <input 
+        id="twoFour" 
+        type="radio"
+        required
+        value="two_four"
+        name="groupSizeValue"
+        class="form-input-radio">
+
+        <label 
+        for="twoFour"
+        class="form-label-input">
+        2 - 4</label>
+      </div>
+
+      <div class="form-size-group">
+        <input 
+        id="fiveTen" 
+        type="radio"
+        required
+        value="five_ten"
+        name="groupSizeValue"
+        class="form-input-radio">
+
+        <label 
+        for="fiveTen"
+        class="form-label-input">
+        5 - 10</label>
+      </div>
+          
+      <div class="form-size-group">
+        <input 
+        id="tenTwenty" 
+        type="radio"
+        required
+        value="ten_twenty"
+        name="groupSizeValue"
+        class="form-input-radio">
+
+        <label 
+        for="tenTwenty"
+        class="form-label-input">
+        10 - 20</label>
+      </div> -->  
```

Comment:
In general, we recommend removing unused code but if you think you will implement it at a later point - that's alright